src/charts/raster-chart.js

```diff
@@ -697,18 +696,18 @@ export default function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
     callback,
     fetchEvenIfEmpty = false
   ) {
+    if (!point) {
+      return
+    }
+
     const height =
       typeof _chart.effectiveHeight === "function"
         ? _chart.effectiveHeight()
         : _chart.height()
     const pixelRatio = _chart._getPixelRatio() || 1
-    const pixel = new HeavyConnect.TPixel({
+    const pixel = {
```

Comment:
can just set the pixel = `{x: ..., y: ...}` - this will get properly converted to a `TPixel` in the `getResultRowForPixelAsync` call below. See: https://github.com/heavyai/heavyai-connector/blob/master/src/heavy-con-es6.js#L1803